build/build.prog.mk

```diff
@@ -1,13 +1,18 @@
+# vim: tabstop=8 shiftwidth=8 noexpandtab:
+#
 # This is a common makefile for various simple usermode programs that reside in
 # subdirectories. This is mostly a template for program-specific
 # makefiles. Generally custom user programs will want to provide their own
 # makefile, but for our own purposes this template is very convenient.
-
-# This template assumes following make variables are set:
-#  PROGRAM: The name for the resulting userspace ELF file. Generally, this will
+#
+# This following make variables are assumed to be set:
+# -PROGRAM: The name for the resulting userspace ELF file. Generally, this will
 #  be the program name installed into /bin directory.
-#  SOURCES: C or assembly files to compile. Can be omitted if there's only one
-#  file to compile named $(PROGRAM).c
+# -SOURCES: C or assembly files to compile. Defaults to $(PROGRAM).c.
+# -BINDIR: The path realtive to $(SYSROOT) at which $(PROGRAM) will be 
+#  installed. Defaults to $(SYSROOT)/bin.
```

Comment:
If it's already relative to `$(SYSROOT)`, then shouldn't the default value be just `bin`?

Reply:
Yes, it should. Done.